### ISSUE TITLE
Width issue on mobile (Search page)

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeUKGenealogy#30]
![Screenshot_20180810-115529.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5728dc91cdca690e00eef9ad/125516c66e82c728ace21e0d46db978826878dba87e6ab7cf60da1cc791e76236455f27e25d7ba6af3006b1300051bff054a5556f9fe8c2dafe06e22de5d59bbd86f0c76c63d14feba824abe13a632c91edbe2e3d7158abd7f37ed998e48341f001b8af7543da69f4dcd5dff59ac.png)

**Comment by arswright-alt-uk** [comment]
@DeniseColbert For future reference, the following process will help to ensure we define and correct issues of this variety:

1. Open Google Chrome
2. Navigate to the page where the issue appears
3. Press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>i</kbd> to open Chrome Developer Tools
4. Confirm that device emulation is toggled
5. Use the dropdown menu or the gray bars at the top of the page to set the viewport width
6. Go to the Menu icon (three dots) in the upper right-hand corner and select **Capture screenshot** or **Capture full size screenshot**
7. Attach the image with a concise description of the issue

(In this case I can see that the Search button is truncated but it was not immediately apparent what the issue was from the title)

**Comment by DeniseColbert** [comment]
Did that, but can't see where *Capture screenshot* is:
![image](https://user-images.githubusercontent.com/19173257/49216279-9abbbd00-f3c2-11e8-8ce8-088052e91d18.png)

(I guess screenshots through the DevTool are more helpful as you get some code info from it, but is the issue here that I didn't put enough description in the title?)

**Comment by arswright-alt-uk** [comment]
@DeniseColbert Ah my bad - forgot that the default dev panel layout puts the dev panel's options at the upper right-hand corner - try the three dots at the upper right-hand corner of the page display panel (next to the responsive/mobile emulation dropdown).

**Comment by DeniseColbert** [comment]
Found the capture menu; thanks.

Right, that makes sense, I'll try to make sure I am more descriptive of the issue. So in this instance, would "Search button truncated on Search page on mobile" be sufficient, with a screen grab?

**Comment by arswright-alt-uk** [comment]
@DeniseColbert 

> So in this instance, would "Search button truncated on Search page on mobile" be sufficient, with a screen grab?

Yes, that description would leave no room for confusion on my part and the screenshot would allow me to quickly replicate the issue - bonus points if you include the viewport width (as this behavior can and will vary between widths with CSS media queries) e.g. "https://freeukgenealogy.org.uk/search - Search button truncated at 320px viewport width" + the screenshot provided would be the ideal description so far as I'm concerned.

**Comment by DeniseColbert** [comment]
Ok to remove gutter
